### PR TITLE
Implement column alignment for cocoaui on medialib branch

### PR DESCRIPTION
**Pull request by tomjepp**
Implement left, center and right alignment for text in playlist cells.

### CODE REVIEW

File: plugins/cocoaui/Playlist/PlaylistViewController.h [hunk 1]
```diff
@@ -25,12 +25,19 @@
 #import "PlaylistView.h"
 #import "ConverterWindowController.h"
 
+typedef NS_ENUM(NSInteger, PlaylistColumnAlignment) {
+    ColumnAlignmentUnknown = -1,
```
Comment (Alexey-Yakovenko) [inline plugins/cocoaui/Playlist/PlaylistViewController.h:29]: This should not be allowed. When the value is loaded from file, it should be converted into representable range of the enum, with ColumnAlignmentLeft being the default.

**Review by Alexey-Yakovenko — CHANGES_REQUESTED** [review]

File: plugins/cocoaui/Playlist/PlaylistViewController.h (hunk 1 above)
Comment (tomjepp) [reply plugins/cocoaui/Playlist/PlaylistViewController.h:29]: This is the default value that's set by the add new column UI now. I can change that to left if you'd prefer?

Comment (tomjepp) [reply plugins/cocoaui/Playlist/PlaylistViewController.h]: I've implemented that.

**Comment by Alexey-Yakovenko** [comment]
Thanks, merged!

**Pull request merged**